peachjam/static/js/judgment_duplicates.js

```diff
@@ -0,0 +1,97 @@
+const debounce = (func, wait, immediate) => {
+  let timeout;
+  return function () {
+    const context = this; const args = arguments;
+    const later = function () {
+      timeout = null;
+      if (!immediate) func.apply(context, args);
+    };
+    const callNow = immediate && !timeout;
+    clearTimeout(timeout);
+    timeout = setTimeout(later, wait);
+    if (callNow) func.apply(context, args);
+  };
+};
+
+function getCaseNumbers () {
+  const visible = {};
+  const hidden = {};
+  const inputs = document.querySelectorAll("input[name^='case_numbers-'], select[name^='case_numbers-']");
```

Comment:
@longhotsummer I actually learned that you can pass multiple selectors to this function and it will bundle everything into one array :smile_cat: